br/cmd/br/backup.go

```diff
@@ -110,6 +108,8 @@ func NewBackupCommand() *cobra.Command {
 			build.LogInfo(build.BR)
 			utils.LogEnvVariables()
 			task.LogArguments(c)
+			// Do not run stat worker in BR.
+			session.DisableStats4Test()
```

Comment:
keep the same as https://github.com/pingcap/tidb/blob/4d9c645933904b044d91796deb1214d9e718f438/br/cmd/br/restore.go#L148